sites/target.py

```diff
@@ -1,36 +1,43 @@
 from selenium import webdriver
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.support.ui import WebDriverWait as wait
-from selenium.webdriver.chrome.options import Options
+# from selenium.webdriver.chrome.options import Options
 from webdriver_manager.chrome import ChromeDriverManager
 from chromedriver_py import binary_path as driver_path
+from selenium.webdriver.firefox.options import Options
+
 from utils import random_delay, send_webhook, create_msg
 from utils.selenium_utils import change_driver
-import settings, time
+import settings, time, random
+from test import mypath
 
-options = Options()
+# options = Options()
```

Comment:
I would go ahead and remove the commented code. Seems like firefox is what will need to be used for Target functionality. We can add multi-browser functionality at a higher level across all classes later on if we decide to stick to a pattern at a later date.

Reply:
I left it in as I was unsure if firefox supported the same type and implementation of preferences to reduce page load times.